src/results/resultUtils.ts

```diff
@@ -462,6 +462,7 @@ export function convertValue<T extends RelTypedValue>(
 export function getDisplayValue(
   typeDef: RelTypeDef,
   value: RelTypedValue['value'],
+  quoteStrings = false,
```

Comment:
TODO: duplicate this whole function? https://relationalai.slack.com/archives/C02C9AV2LL9/p1661370165042819?thread_ts=1661274493.782329&cid=C02C9AV2LL9